syn/syn.ys

```diff
@@ -0,0 +1,18 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# yosys synthesis script
+
+read -sv prim_clock_gating.v syn_out/*.v
+synth -top ibex_core
+write_verilog ibex_core_premap.v
+
+# mapping to cmos_cells.lib
+dfflibmap -liberty cmos_cells.lib
```

Comment:
The cmos_cells.lib whilst fine for demonstration purposes or exploring how yosys works is pretty useless for actual synthesis.  Should be fine for this initial PR but we do need to source a better library that we can use.

Reply:
Yes, absolutely. At a minimum, this library should include a MUX2 and also an ICG. 

Do you know of a good open-source std. cell library? Yosys documentation has a link to some but not sure which one is good.